src/wp-includes/functions.php

```diff
@@ -4123,7 +4123,7 @@ function _deprecated_hook( $hook, $version, $replacement = null, $message = null
  *
  * @param string $function The function that was called.
  * @param string $message  A message explaining what has been done incorrectly.
- * @param string $version  The version of WordPress where the message was added.
+ * @param string $version  The version of ClassicPress where the message was added.
```

Comment:
Let's say "The version of ClassicPress or WordPress" for now.  We'll prefix these version numbers with `WP-` if they are WordPress versions.  See #27.